tests/ui/specs/Popup/Popup-specs.js

```diff
@@ -862,9 +880,10 @@ describe('Popup', function () {
 				Page.waitForOpen(popup);
 
 				expectOpen(popupCommon);
+				Page.waitForFocused(popup.buttonOK);
 
-				Page.spotlightDown(); // OK button must have focus after Popup opens. This should be fixed and this line removed.
 				Page.spotlightRight();
+				Page.waitForFocused(popup.buttonCancel);
 				Page.spotlightDown();
 
 				expect(popup.buttonCancel.isFocused()).to.be.true();
```

Comment:
Here also, we need to change the expectation and description.
```suggestion
				Page.waitForFocused(popupCommon.buttonPopup8);

				expect(popupCommon.buttonPopup8.isFocused()).to.be.true();
```